src/ui/actions/app.ts

```diff
@@ -202,6 +203,56 @@ export function loadMouseTargets(): UIThunkAction {
   };
 }
 
+export function setBreakpointsFromClipboard(): UIThunkAction {
+  return async (dispatch, getState, { ThreadFront }) => {
+    const currentFocusedElement = document.activeElement;
+    let text = "";
+    try {
+      // The `readText()` API requires that the current DOM be focused somehow
+      document.body.focus();
+      text = await navigator.clipboard.readText();
+
+      // Was the clipboard data even valid JSON?
+      const parsedValue = JSON.parse(text);
+      // Assuming it was JSON, does it appear to be an array of Point objects?
+      if (Array.isArray(parsedValue) && parsedValue.every(isValidPoint)) {
+        // TODO [FE-874] This will need to change if we modify how we do per-recording persistence
+
+        // This value appears to be actual breakpoints. Our current implementation watches
+        //  for changes in `localStorage` and will auto-update itself when that event fires.
+        const storageKey = `${ThreadFront.recordingId!}::points`;
+        // Just reuse the text from the clipboard
+        localStorage.setItem(storageKey, text);
```

Comment:
Would it be much harder to _merge_ these? Seems like we could do it by comparing locations?

Might be annoying if someone sent you their log points and it clobbered all of your existing ones.

Reply:
It's possible. My mental use case atm is the scenario where you and I were looking at the `<BreakpointNavigation>` bug last week - I'd spent a day building up a set of print statements, you were about to look for the first time, and I could have potentially sent over my list as a starting point for you to paste in and then tweak further.

If we really want me to tweak this further I can, but my goal here was just get the MVP in with minimal effort and time spent.